packages/core/.storybook/preview-body.html

```diff
@@ -0,0 +1,3 @@
+<iframe title="Stencil Dev Server Connector 2.3.0 ⚡" src="/~dev-server"
```

Comment:
Dit is de development injectie van Stencil, 1 op 1 gekopieerd uit een runtime `index.html`